src/pickBy.test.ts

```diff
@@ -0,0 +1,20 @@
+import { pickBy } from './pickBy';
+import { pipe } from './pipe';
+
+describe('data first', () => {
+  test('it should pick props', () => {
+    const result = pickBy({a: 1, b: 2, A: 3, B: 4}, (val, key) => key.toUpperCase() === key);
+    expect(result).toStrictEqual({ A: 3, B: 4 });
+  });
+  test('allow undefined or null', () => {
+    expect(pickBy(undefined as any, (val, key) => key === 'foo')).toEqual({});
+    expect(pickBy(null as any, (val, key) => key === 'foo')).toEqual({});
+  });
```

Comment:
why would we want to allow null or undefined as input? None of the other object methods do this...

Reply:
I made same as https://github.com/remeda/remeda/blob/master/src/pick.test.ts#L10. Null/undefined disabled by types but covered with tests.

Reply:
oh I didn't know that `pick` also allows it 👍